### PR TITLE
Type script

### LINKED ISSUE #10: Apps version
Add a endpoint on bb-bot to add version for apps on format:

/version/app/type_deploy/version

app = name of the app as on /services
type_deploy = env where the app is deployed(dev, homolog, prod, pil)

version = number of the version

### PR DESCRIPTION
**Pull request by gutorc92**
Resolves #10 and #9

### CODE REVIEW

File: src/routers/kbRouters.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    console.log('error', err)
+  }
+})
+
+router.post('/deploy/promster', async (req, res) => {
+  try {
+    const kubeconfig = new KubeConfig()
+    kubeconfig.loadFromFile(KB_CONFIG)
+
+    // kubeconfig.loadFromString(JSON.stringify(config))
```
Comment (abilioesteves) [inline src/routers/kbRouters.js:37]: @gutorc92, you can move this logic outside this code. It is a duplicate of lines 14-17

File: src/routers/kbRouters.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    })
+  } catch (err) {
+    console.log('error', err)
+  }
+})
+
+router.get('/check/deploy/:app', async (req, res) => {
+  try {
+    const kubeconfig = new KubeConfig()
+    kubeconfig.loadFromFile(KB_CONFIG)
```
Comment (abilioesteves) [inline src/routers/kbRouters.js:61]: @gutorc92, once again duplicated

File: src/types/App.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      {
+        name: 'SCRAPE_ETCD_PATH',
+        value: `/services/${this._name}`
+      }
+    ]
+    const remoteWrite = false
+    if (remoteWrite) {
+      envVars.push({
+        name: 'REMOTE_WRITE_URL',
+        value: 'http://nginx.cortex.svc.cluster.local/api/prom/push'
```
Comment (abilioesteves) [inline src/types/App.ts:137]: @gutorc92, I don't think this should be a fixed value.

File: src/types/App.ts [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+      } else if (typeof data.ips === 'string') {
+        this.setIps(data.ips)
+      }
+    }
+  }
+
+  public getManifest () : object {
+    const deploymentManifest = require('../deployments/bb-promster.json')
+    const deploy = { ...deploymentManifest }
+    const etcdService = 'http://etcd.barimetrics.svc.cluster.local:2379'
```
Comment (abilioesteves) [inline src/types/App.ts:102]: @gutorc92, I don't think this should be a fixed value.

File: src/deployments/bb-promster.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                "name": "ETCD_URLS",
+                "value": "http://etcd.default.svc.cluster.local:2379"
+              },
+              {
+                "name": "SCRAPE_ETCD_PATH",
+                "value": "/services/example"
+              },
+              {
+                "name": "REMOTE_WRITE_URL",
+                "value": "http://nginx.cortex.svc.cluster.local/api/prom/push"
```
Comment (abilioesteves) [inline src/deployments/bb-promster.json:60]: @gutorc92, I'm worried about these fixed values here. What are you using this `bb-promster.json` for?

**Comment by abilioesteves** [comment]
Also, @gutorc92, I think space identation are awful. Could you please make sure the core is idented with tabs instead?

File: src/deployments/bb-promster.json (hunk 5 above)
Comment (gutorc92) [reply src/deployments/bb-promster.json]: bb-promster.json is a template for a deployment on Kubernetes.

**Comment by gutorc92** [comment]
> Also, @gutorc92, I think space identation are awful. Could you please make sure the core is idented with tabs instead?

it is done.

File: README.md [hunk 6]
```diff
@@ -19,10 +19,13 @@ A repository to define a Telegram Bot for handling alerts emitted by a Big Broth
 
 7. Now go to the bot on Telegram, and add a new App. Inform the App name (e.g. `Example`) and the app address (e.g. `example-bb-promster:9090`). You'll be automatically subscribed to the app you've just added.
 
+
 The example client app `bb-promster` cluster will get registered to the **Big Brother's** ETCD and **Big Brother** will then start collecting metrics by federating it.
 
 Also, access `http://localhost:3000/test` on your browser to dispatch test alerts and see if you get them at your Telegram chat. 
 
+If you are using VSCode, you should use **editor.insertSpaces** equal **false** on your settings.
```
Comment (abilioesteves) [inline README.md:27]: @gutorc92, I would remove this. VSCode has the `editor.detectIdentation` config just for that.

File: src/agent.js [hunk 7]
```diff
@@ -2,179 +2,182 @@
...
-    __("Remove an app"),
-    __("Change an app's bb-promster address"),
-    __("Help with setting up my app observation cluster")
-];
+	__('List all apps being monitored'),
+	__('Subscribe to alerts'),
+	__('Unsubscribe to alerts'),
+	__('Add a new app'),
+	__('Remove an app'),
+	__("Change an app's bb-promster address"),
```
Comment (abilioesteves) [inline src/agent.js:27]: @gutorc92, a little bit of a OCD here, but why all other string are with simple quotes and this one is with double quotes?

**Pull request merged**